### PR TITLE
Added establishment types reference data

### PR DESCRIPTION
**Pull request by rmbielby**
## Overview of changes

Similar to the SEN data guidance update, this presents the first iteration of standardisation of the school type fields as reviewed by the data harmonisation champions.

## Why are these changes being made?

These are part of the ongoing effort to establish standardised data guidance for publication teams to follow.

## Detailed description of changes

Standard field names and items are outlined for:

Establishment type group
Establishment governance
Education phase
Establishment type
Provider type

## Checklist before requesting a review
- [ ] I have checked the contributing guidelines
- [ ] I have checked for and linked any relevant issues that this may resolve
- [ ] I have checked that these changes build locally
- [ ] I understand that if merged into main, these changes will be publicly available

### CODE REVIEW

File: statistics-production/ud.qmd [hunk 1]
```diff
@@ -1044,6 +1044,71 @@ Where there is a reasonable need to publish or report statistics for a full aggr
...
+---
+
+### Establishment governance
+
+---
+
+The **establishment_governance** field provides an opportunity to list the governance of an establishment, e.g. Academy, Non-maintained, Independent etc. It can be assigned the **establishment_type_group** as a filter grouping column.
+
+| col_name | Breakdown topic entry | Filter item |
+| -------- | --------------------- | ----------- |
```
Comment (cjrace) [inline statistics-production/ud.qmd:1075]: Same comment for this table

File: statistics-production/ud.qmd [hunk 2]
```diff
@@ -1044,6 +1044,71 @@ Where there is a reasonable need to publish or report statistics for a full aggr
...
+---
+
+### Education phase
+
+---
+
+Education phase should be listed under he column name **education_phase** where given as a standalone filter or Education phase if included in the **breakdown_topic** / **breakdown** combination. When included as an individual filter, it can be attributed **establishment_type_group** as the filter grouping column where available.
+
+| col_name | Breakdown topic entry | Filter item | School census code |
+| -------- | --------------------- | ----------- | ------------------ |
```
Comment (cjrace) [inline statistics-production/ud.qmd:1094]: And same comment for this table too

File: statistics-production/ud.qmd [hunk 3]
```diff
@@ -1044,6 +1044,71 @@ Where there is a reasonable need to publish or report statistics for a full aggr
...
+
+A standardised set of establishment type fields has been developed, largely based on the data reported by the School and Pupils Statistics publication. Reference files for the standardised fields are available in the data screener GitHub repository, with the standards summarised below.
+
+---
+
+### Establishment type group
+
+---
+
+| col_name | Breakdown topic entry | Filter item |
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply statistics-production/ud.qmd]: Ok, given that switch a go

File: statistics-production/ud.qmd [hunk 4]
```diff
@@ -1044,6 +1044,89 @@ Where there is a reasonable need to publish or report statistics for a full aggr
...
+| Academy |
+| All state funded |
+| Independent |
+| Local authority maintained |
+| Non-maintained |
+| Total |
+
+:::
+
+If needed, these items can also be reported in the standard **breakdown** field, using a **breakdown_topic** of Establishment type group.
```
Comment (cjrace) [inline statistics-production/ud.qmd:1070]: Do we have a section of guidance on this to link to higher up the page?

File: statistics-production/ud.qmd [hunk 5]
```diff
@@ -1044,6 +1044,89 @@ Where there is a reasonable need to publish or report statistics for a full aggr
...
+| Foundation |
+| Independent |
+| Non-maintained |
+| Voluntary aided |
+| Voluntary controlled |
+| Total |
+
+:::
+
+If needed, these items can also be reported in the standard **breakdown** field, using a **breakdown_topic** of Establishment governance.
```
Comment (cjrace) [inline statistics-production/ud.qmd:1097]: Do we have a section of guidance on this to link to higher up the page?

**Review by cjrace — CHANGES_REQUESTED** [review]
Like this now I think that's much cleaner to understand at a glance, one minor point about in page links then will be happy to approve

File: statistics-production/ud.qmd [hunk 6]
```diff
@@ -1044,6 +1044,89 @@ Where there is a reasonable need to publish or report statistics for a full aggr
...
+
+---
+
+### Education phase
+
+---
+
+Education phase should be listed under he column name **education_phase** where given as a standalone filter or Education phase if included in the **breakdown_topic** / **breakdown** combination. When included as an individual filter, it can be attributed **establishment_type_group** as the filter grouping column where available.
```
Comment (cjrace) [inline statistics-production/ud.qmd:1105]: he typo!

**Review by cjrace — APPROVED** [review]

**Pull request merged**